frontend/src/bundles/home/components/main-content/main-content.tsx

```diff
@@ -0,0 +1,16 @@
+import { Box } from '@chakra-ui/react';
```

Comment:
if you import components outside common bundle, it should imported from components.ts